server/utils/applicationUtils.ts

```diff
@@ -20,4 +22,33 @@ const taskLink = (task: TaskNames, id: string): string => {
   })}" aria-describedby="eligibility-${task}" data-cy-task-name="${task}">${taskLookup[task]}</a>`
 }
 
-export { getTaskStatus, taskLink }
+const getService = (req: Request): 'approved-premises' | 'temporary-accommodation' => {
```

Comment:
Non-blocking suggestion: There's a fair bit of conditional logic going on in one method, there might be an opportunity to refactor this into some private methods. That said it's readable and unlikely to get changed much more. This approach may be more common in the Typescript world in which case I'll get back in my hole :)